file_system/test_manifest_view.py

```diff
@@ -26,26 +26,25 @@ class TestManifest(TestCase):
         "28ca34e8-9d4c-4543-9fc7-981bf5f6a97f.files.json",
```

Comment:
Can we move this test under `file_system/tests/test_manifest_view.py`, and also move `tests.py` (also rename it in something like tests_file_system.py) under the same directory. It is a good practice when having multiple test files to keep them in `tests` package in the app, like we did here https://github.com/mskcc/beagle/tree/master/runner/tests

Reply:
I believe I've resolved these issues in the most recent commit. I double check that the changes work on dev as well. Let me know your thoughts!